pyproject.toml

```diff
@@ -1,35 +1,35 @@
 [tool.poetry]
 name = "political_party_analysis"
-version = "0.1.0"
+version = "0.2.0"
 description = ""
 authors = ["Iswariya Manivannan <iswariya.m17@gmail.com>"]
 
 [tool.poetry.dependencies]
-python = ">=3.7.1,<3.10"
-requests = "^2.26.0"
-numpy = "^1.21.2"
-pandas = "^1.3.2"
-matplotlib = "^3.4.3"
-scikit-learn = "^1.0"
+python = ">=3.9,<3.13"
+requests = "*"
+numpy = "*"
+pandas = "*"
+matplotlib = "*"
+scikit-learn = "*"
 
 [tool.poetry.dev-dependencies]
-pytest = "^5.2"
-black = "^21.8b0"
-flake8 = "^3.9.2"
-mypy = "^0.910"
-pytest-mock = "^3.6.1"
+pytest = "*"
+black = "*"
+flake8 = "*"
+mypy = "*"
+pytest-mock = "*"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.mypy]
-python_version = "3.7"
+python_version = "3.8"
```

Comment:
You should also update the versions mentioned in README

Reply:
ah yes, good catch! I also made some other minor README improvements while I was in there.